spec/models/file_search_spec.rb

```diff
@@ -13,16 +13,35 @@
   let(:catalog_query) { double("Catalog query", search_params_logic: logic) }
   let(:file_query) { double("File query") }
   let(:files) { [double("GenericFile1"), double("GenericFile2")] }
+  let(:response) { double("Response",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.